packages/suite/src/views/wallet/receive/components/UsedAddresses.tsx

```diff
@@ -226,37 +162,42 @@ export const UsedAddresses = ({
     const actionHideVisible = limit > DEFAULT_LIMIT;
 
     return (
-        <Card margin={{ bottom: spacings.xxxl }} overflow="hidden" paddingType="large">
-            <Column>
-                <GridTable>
-                    <HeaderItem>
-                        <Translation id="RECEIVE_TABLE_ADDRESS" />
-                    </HeaderItem>
-                    <HeaderItem />
-                    <HeaderItem $align="right">
-                        <Translation id="RECEIVE_TABLE_RECEIVED" />
-                    </HeaderItem>
-
-                    {list.slice(0, limit).map((addr, index) => (
-                        <Item
-                            index={index}
-                            key={addr.path}
-                            addr={addr}
-                            symbol={account.symbol}
-                            locked={locked}
-                            metadataPayload={{
-                                type: 'addressLabel',
-                                entityKey: account.key,
-                                defaultValue: addr.address,
-                                value: addressLabels[addr.address],
-                            }}
-                            onClick={() => dispatch(showAddress(addr.path, addr.address))}
-                        />
-                    ))}
-                </GridTable>
+        <Card margin={{ bottom: spacings.xxxl }} overflow="hidden" paddingType="normal">
+            <Column alignItems="stretch" gap={spacings.md}>
+                <Table>
+                    <Table.Header>
+                        <Table.Row>
+                            <Table.Cell>
+                                <Translation id="RECEIVE_TABLE_ADDRESS" />
+                            </Table.Cell>
+                            <Table.Cell />
+                            <Table.Cell align="right">
+                                <Translation id="RECEIVE_TABLE_RECEIVED" />
+                            </Table.Cell>
+                        </Table.Row>
+                    </Table.Header>
+                    <Table.Body>
+                        {list.slice(0, limit).map((addr, index) => (
+                            <Item
+                                index={index}
```

Comment:
Can we replace `index` with `addr.address`?

Reply:
What's the reason? It's used for test ids and I would have to update E2E tests?

Reply:
OK, keep it